### DISCUSSION TITLE
How can i use prior to specific the beta parameter of a channel variable?

### DISCUSSION THREAD

**Discussion by Brites101** [pymc-labs/pymc-marketing#511] [Q&A]
In pymc-marketing i am able to set the prior distributions for the beta of each channel variable, alongside their adstock and diminishing return. 

I understand how to describe prior knowledge of adstock and diminishing return of these channel variables, by setting a distribution with desired mean and lower variance.

However, i fail to grasp how i can set the beta parameters of a channel using their prior. Given the beta parameters as estimated using a HalfNormal distribution, i can skew it closer/farther to zero, but i don't see how i could set a specific value for beta. 

Can i use the prior of the beta variable of a specific channel to set a value of beta that i am confident in?

**Comment by BahgatN** [comment]
I'm new to pymc_marketing, but I've encountered a similar issue. It seems that the HalfNormal distribution, which is typically considered a weakly informative prior, may not be the best choice for certain scenarios, as it restricts the ability to decrease variance or increase precision for higher values of the mean (impact). Instead, adjustments can only be made for lower mean values in the HalfNormal distribution to have higher precision.

pymc uses the MaxAbs Scalar, so that one potential solution I found is to switch to a beta distribution as a prior for beta_channel, using a parameterization for shape and scale (e.g alpha = 20 and beta = 20). However, I noticed when doing so, this change could impact convergence and sampler performance speed to be very slow
 
See[ Changing beta channel prior distributions from weakly to informative](https://github.com/pymc-labs/pymc-marketing/discussions/588)

**Comment by cetagostini** [comment]
Hey @Brites101 

You can check the info here and get inspiration about how to select priors: https://github.com/pymc-labs/pymc-marketing/discussions/617#discussioncomment-9244139

Let me know if helps!
